frontend/.env.local

```
@@ -2,4 +2,6 @@
 
 REACT_APP_MS_CLIENT_ID=18503d32-786a-40cf-b3b4-5abe6fd66666
 REACT_APP_MS_REDIRECT_URI=http://localhost:3001
-REACT_APP_DEPLOYMENT=local
\ No newline at end of file
+REACT_APP_DEPLOYMENT=local
+
+REACT_APP_STREAM_API_KEY='94ggnkahafdr'
```

Sorry for nitpicking here but we should keep our style as consistent as possible i.e. removing the newline and the '
```suggestion
REACT_APP_STREAM_API_KEY=94ggnkahafdr
```